### PR TITLE
Add foreman_fog_proxmox plugin in deb packaging

### PR DESCRIPTION
**Pull request by tristanrobert**
Add [foreman_proxmox plugin](https://github.com/tristanrobert/foreman_proxmox).
[Gem](https://rubygems.org/gems/the_foreman_proxmox) is published.

For plugin updates, please indicate which repos this should be built into:

* [ ] Nightly
* [ ] 1.18
* [x] 1.17
* [ ] 1.16

See Foreman's [plugin maintainer documentation](https://projects.theforeman.org/projects/foreman/wiki/How_to_Create_a_Plugin#Release-strategies) for more information.

---

### CODE REVIEW

File: plugins/ruby-foreman-proxmox/debian/changelog [hunk 1]
```diff
@@ -0,0 +1,5 @@
+ruby-foreman-proxmox (0.0.1) unstable; urgency=low
```
Comment (mmoll) [inline plugins/ruby-foreman-proxmox/debian/changelog]: `0.3.1-1`, please :)

File: plugins/ruby-foreman-proxmox/debian/control [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+Section: ruby
+Priority: extra
+Maintainer: Tristan Robert <tristan.robert.44@gmail.com>
+Build-Depends: debhelper (>= 8.0.0), foreman-assets (>= 1.17.0), foreman (>= 1.17.0), foreman-sqlite3 (>= 1.17.0)
+Standards-Version: 3.9.3
+Homepage: https://github.com/tristanrobert/foreman_proxmox
+
+Package: ruby-foreman-proxmox
+Architecture: all
+Depends: ${misc:Depends}, bundler, foreman (>= 1.17.0)
```
Comment (mmoll) [inline plugins/ruby-foreman-proxmox/debian/control]: I *guess* `foreman-compute` is also needed, as this brings the fog gem.

**Comment by tristanrobert** [comment]
Oops! I'll fix it.

**Comment by ekohl** [comment]
Similar as in the RPM: would it be a good time to rename the gem to `foreman_proxmox` to follow the naming convention?

**Comment by tristanrobert** [comment]
Sorry but the gem [foreman_proxmox](https://rubygems.org/gems/foreman_proxmox) is already used, that's why I have to rename it the_foreman_proxmox.
Do you have any idea how I could rename  it?

**Comment by ekohl** [comment]
Ah, that's tricky. You'd probably need to contact the owner. The email is listed on https://github.com/fiesensee. In the mean time I'd recommend choosing a name with the `foreman_` prefix. `foreman_proxmox2`? If you've played Borderlands then `foreman_proxmoxxie` could be nice.

**Comment by tristanrobert** [comment]
Sorry I don't know the game borderlands. I think about foreman_fog_proxmox because it uses fog and the other gem don't. Is it ok for you? If so I'll rename it definitively.

**Comment by ekohl** [comment]
I think `foreman_fog_proxmox` would work.

**Comment by tristanrobert** [comment]
@ekohl Ihave renamed it as foreman_fog_proxmox

**Comment by tristanrobert** [comment]
Replace by new one

**Pull request closed without merging**